dist/tools/ethos/start_network.sh

```diff
@@ -1,15 +1,23 @@
 #!/bin/sh
 
-ETHOS_DIR="$(dirname $(readlink -f $0))"
+ETHOS_DIR="$(dirname $(readlink -f "$0"))"
```

Comment:
shellcheck suggests that `$(readlink -f "$0")` should also be quoted. The current construct breaks if there is whitespace in the path returned by `readlink`